config/jobs/kubernetes/kops/build_jobs.py

```diff
@@ -987,6 +979,7 @@ def generate_presubmits_e2e():
             cloud="aws",
             distro="deb11",
             k8s_version="ci",
+            irsa=True,
```

Comment:
There is no need to include these.

Reply:
There is, take a look at how pull requests are handled, irsa defaults to False.